dists/install.sh

```diff
@@ -46,10 +46,9 @@ main() {
   uname -a | grep -qi 'Darwin' ; is_darwin=$?
 
   os=$(uname -s | tr '[A-Z]' '[a-z]')
-  ext=zip
+  ext='tar.gz'
```

Comment:
The switch case after this line could be simplified in a single if, don't you think?